### ISSUE TITLE
[V3 Downloader] Error when adding repo multiple times

### ISSUE THREAD

**Issue by aikaterna** [Cog-Creators/Red-DiscordBot#1671]
# Command bugs

#### Command name

[p]repo add

#### What cog is this command from?

Downloader

#### What were you expecting to happen?

A message in Discord saying "That repo name you provided already exists. Please choose another."

#### What actually happened?

```
Exception in command 'repo add'
Traceback (most recent call last):
  File "/home/aikaterna/v3/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/aikaterna/v3/lib/python3.6/site-packages/redbot/cogs/downloader/downloader.py", line 228, in _repo_add
    repo = await self._repo_manager.add_repo(name=name, url=repo_url, branch=branch)
  File "/home/aikaterna/v3/lib/python3.6/site-packages/redbot/cogs/downloader/repo_manager.py", line 538, in add_repo
    "That repo name you provided already exists." " Please choose another."
redbot.cogs.downloader.errors.InvalidRepoName: That repo name you provided already exists. Please choose another.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/aikaterna/v3/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/home/aikaterna/v3/lib/python3.6/site-packages/discord/ext/commands/core.py", line 927, in invoke
    yield from ctx.invoked_subcommand.invoke(ctx)
  File "/home/aikaterna/v3/lib/python3.6/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/home/aikaterna/v3/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: InvalidRepoName: That repo name you provided already exists. Please choose another.
```

#### How can we reproduce this issue?

Add a repo, then add it again with the same name.

**Comment by tekulvw** [comment]
You didn't see the existing name message in discord at all?

**Comment by aikaterna** [comment]
The message in Discord was the standard "Error in command 'repo add'. Check your console or logs for details."

**Comment by tekulvw** [comment]
Fixed with #1692
